src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  position: absolute;
+  right: 50px;
+  top: 22px;
+}
+
+.nav__link {
+  color: black;
+  padding-top: 20px;
+  padding-bottom: 20px;
+}
+
+.nav__item {
+  display: inline-block;
+  margin-left: 15px;
+  font-family: Roboto , sans-serif;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  position: relative;
+
+}
+
+.logo {
+  margin-left: 50px;
+  margin-top: 10px;
+}
+
+.is-active::after {
+  color: yellow;
+}
```

Comment:
Why yellow? Haven't seen it on design.

Reply:
i was trying smt by myself i forgot it there ))